notebook/services/api/api.yaml

```diff
@@ -814,6 +814,10 @@ definitions:
         type: string
         description: Last modified timestamp
         format: dateTime
+      size:
+        type: integer
```

Comment:
Integer or None ? If size can't be found, probably say that in the description at least.